pkg/api/server.go

```diff
@@ -87,9 +88,13 @@ func (s *Server) startGRPC() error {
 	stream = append(stream, telemetryInterceptor.Stream())
 
 	if s.Config.Authn.Enable {
+		limiter := ratelimiter.NewTokenBucketRateLimiter(s.Log)
+		// Expire buckets after 6 hours of inactivity,
+		// sweep for expired buckets every hour.
+		go limiter.Janitor(s.ctx, time.Hour, 6*time.Hour)
```

Comment:
Not quite sure what the settings should be. Thoughts?

Reply:
Given that we've invested all this effort making it non-blocking, I'd suggest doing it more frequently (maybe every 30 mins) to keep memory usage lower

Reply:
OK, how about this. Given that a bucket is refilled in 5 minutes given our current settings, how about expiration 1 hour and sweep every 10 mins? Might help keeping the maps smaller as well.